store/database.go

```diff
@@ -68,11 +68,11 @@ type VersionedDatabase interface {
 // Committer defines an API for committing state.
 type Committer interface {
 	WriteBatch(cs *Changeset) error
-	WorkingHash() []byte
-	GetLatestVersion() uint64
+	WorkingStoreInfos(version uint64) []StoreInfo
```

Comment:
why was working hash removed here?

Reply:
it is replaced with WorkingStoreInfos, the root store has still WorkingHash